devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -63,6 +68,10 @@ def parse(String description) {
 def parseAttrMessage(description) {
 	def descMap = zigbee.parseDescriptionAsMap(description)
 	def map = [:]
+	if (isAduroSmartRemote()) {
+		def descMap = zigbee.parseDescriptionAsMap(description)
+		result = parseAduroSmartButtonMessage(descMap)
+    	}
```

Comment:
Reconsider the placement of this `if` statement. I'm worried it might result in errors when the device reports its battery level.

Reply:
And the results will be overwritten when the device sends ONOFF messages.

It might fit as an `else if` after the power config cluster statement

Reply:
@greens 
Thanks.
Fixed.